test/addons-napi/test_exception/test.js

```diff
@@ -44,7 +45,8 @@ const theError = new Error('Some error');
                      ` ${caughtError} was passed`);
 
   // Test that the exception state remains clear when no exception is thrown
-  assert.strictEqual(test_exception.wasPending(), false,
-                     'VM was not marked as having an exception pending' +
-                     ' when none was allowed through');
+  const exception_pending = test_exception.wasPending();
+  assert.strictEqual(exception_pending, false,
+                     'Expected no exception to be pending, but' +
+                     ` it was marked as ${exception_pending}`);
```

Comment:
Same comment as above, although with the text changed to reverse the expectation.